### PR TITLE
Translated XML documentation in Data, Data/Schema and Data/SqlServer

### PR DESCRIPTION
**Pull request by satano**
Contributes to #30 .

### CODE REVIEW

File: Kros.Utils/src/Kros.Utils/Data/IdGeneratorFactories.cs [hunk 1]
```diff
@@ -68,17 +82,15 @@ public static IIdGeneratorFactory GetFactory(DbConnection connection)
...
-        /// Connection string, na základe ktorého sa vytvorí connection pre vykonanie dotazu na získanie identifikátorov.
-        /// </param>
-        /// <param name="adoClientName">
-        /// Názov ado clienta. (napr. pre <see cref="System.Data.SqlClient.SqlConnection"/> to je: System.Data.SqlClient)
-        /// </param>
-        /// <returns>
-        /// Inštancia <see cref="IIdGeneratorFactory"/>.
-        /// </returns>
+        /// <param name="connectionString">Connection string for database.</param>
+        /// <param name="adoClientName">Name, which specifies database type..</param>
```
Comment (LukasSefcik) [inline Kros.Utils/src/Kros.Utils/Data/IdGeneratorFactories.cs]: Two dots at the end.

File: Kros.Utils/src/Kros.Utils/Data/Schema/DefaultValueParsers.cs [hunk 2]
```diff
@@ -5,18 +5,18 @@
 namespace Kros.Data.Schema
 {
     /// <summary>
-    /// Štandardné parsery pre predvolené hodnoty stĺpcov databáz. Predvolené hodnoty sú v databáze uložené ako reťazec
-    /// a je potrebné ich konvertovať na typ daného stĺpca. Ak sa hodnotu nepodarilo skonvertovať na určený typ,
-    /// vždy je vrátená hodnota <c>null</c>.
+    /// Standard parsers for columns' default values. The default values in database are stored as string and it is
```
Comment (LukasSefcik) [inline Kros.Utils/src/Kros.Utils/Data/Schema/DefaultValueParsers.cs]: Bad position of the apostrophe.

File: Kros.Utils/src/Kros.Utils/Data/Schema/ForeignKeySchemaCollection.cs [hunk 3]
```diff
@@ -6,53 +6,49 @@
...
 {
     /// <summary>
-    /// Zoznam cudzích kľúčov tabuľky <see cref="TableSchema"/>.
+    /// List of foreign keys for table <see cref="TableSchema"/>.
     /// </summary>
-    /// <remarks>Cudzím kľúčom pridaným do zoznamu je automaticky nastavená tabuľka <see cref="IndexSchema.Table"/>.
-    /// Do zoznamu nie je možné pridať cudzí kľúč, ktorý už patrí inej tabuľke. V takom prípade je vyvolaná výnimka
-    /// <see cref="InvalidOperationException"/>.</remarks>
+    /// <remarks>To the foreign keys added to this list is automatically set their <see cref="ForeignKeySchema.Table"/>.
+    /// Foreign ey can belong only to one table.</remarks>
```
Comment (LukasSefcik) [inline Kros.Utils/src/Kros.Utils/Data/Schema/ForeignKeySchemaCollection.cs]: Typo, ey -> key.

File: Kros.Utils/src/Kros.Utils/Data/SqlServer/SqlServerIdGenerator.cs [hunk 4]
```diff
@@ -65,13 +75,13 @@ protected override int GetNewIdFromDbCore()
...
         /// <summary>
-        /// Získanie scriptu na vytvorenie store procedure.
+        /// Returns SQL script for creating stored procedure, which generates IDs.
         /// </summary>
         public static string GetStoredProcedureCreationScript() =>
             ResourceHelper.GetResourceContent("Kros.Resources.SqlIdGeneratorStoredProcedureScript.sql");
 
         /// <summary>
-        /// Získanie scriptu na vytvorenie IdStore tabuľky.
+        /// Returns SQL script for creating table in databas to for storing IDs.
```
Comment (LukasSefcik) [inline Kros.Utils/src/Kros.Utils/Data/SqlServer/SqlServerIdGenerator.cs]: Databas -> Database.

File: Kros.Utils/src/Kros.Utils/Data/SqlServer/SqlServerIdGeneratorFactory.cs [hunk 5]
```diff
@@ -35,7 +29,8 @@ public SqlServerIdGeneratorFactory(SqlConnection connection)
         /// <summary>
         /// Initializes a new instance of the <see cref="SqlServerIdGeneratorFactory"/> class.
         /// </summary>
-        /// <param name="connectionString">Connection string, ktorý sa použije na vytvorenie conenction pre získavanie unikátnych identifikátorov.</param>
+        /// <param name="connectionString">Database connection connection.
```
Comment (LukasSefcik) [inline Kros.Utils/src/Kros.Utils/Data/SqlServer/SqlServerIdGeneratorFactory.cs]: Database connection string.

File: Kros.Utils/src/Kros.Utils/Data/Schema/DefaultValueParsers.cs (hunk 2 above)
Comment (satano) [reply Kros.Utils/src/Kros.Utils/Data/Schema/DefaultValueParsers.cs]: It is good. "columns" is plural. ;)

File: Kros.Utils/src/Kros.Utils/Data/IdGeneratorBase.cs [hunk 6]
```diff
@@ -1,78 +1,91 @@
...
+            Check.NotNullOrWhiteSpace(connectionString, nameof(connectionString));
             Connection = CreateConnection(connectionString);
             _disposeOfConnection = true;
         }
 
         private IdGeneratorBase(string tableName, int batchSize)
         {
             TableName = Check.NotNull(tableName, nameof(tableName));
-            BatchSize = Check.GreaterOrEqualThan(batchSize, 0, nameof(tableName));
+            BatchSize = Check.GreaterThan(batchSize, 0, nameof(batchSize));
```
[the remark this answers is not in the document]
Comment (satano) [reply Kros.Utils/src/Kros.Utils/Data/IdGeneratorBase.cs]: What is the logic of having the `batchSize` of value `0`? Even if it works, it is strange. In my opinion, the problem is somewhere else (KORM probablky?).

[the remark this answers is not in the document]
Comment (Burgyn) [reply Kros.Utils/src/Kros.Utils/Data/IdGeneratorBase.cs]: I agree, it's strange. I tried to investigate it and the problem is in KORM. I have created a new issue #65 for it.

**Pull request merged**